src/libs/sfdk/sfdk_version_p.h.in

```diff
@@ -0,0 +1,20 @@
+#ifndef SFDK_VERSION_P_H_IN
+#define SFDK_VERSION_P_H_IN
+
+
+namespace Sfdk {
+namespace Constants {
+
+#ifdef BASE_OS_NAME
+const char OS_NAME[] = BASE_OS_NAME \" OS\";
+const char SDK_NAME[] = BASE_OS_NAME \" SDK\";
+#else
+const char OS_NAME[] = \"Sailfish OS\";
+const char SDK_NAME[] = \"Sailfish SDK\";
+#endif
```

Comment:
Notice the duplicity in handling the defaults here and in sfdk.pro.

After the change suggested under sfdk.pro, this should become

```
const char OS_VARIANT[] = \"$${SFDK_OS_VARIANT}\";
const char SDK_VARIANT[] = \"$${SFDK_SDK_VARIANT}\";
```

But then the question arises how to mark these for translation? In a regular header file this would be done with QT_TRANSLATE_NOOP, but this is just the template and I am not sure if translations.pro invokes lupdate in a way that it will notice the resulting header file.

Reply:
QT_TRANSLATE_NOOP not need this, it will be translated with `tr()` in `Sdk::sdkVariant()`

Reply:
Yes, tr() will translate it. But first the translation needs to be defined and for that the string needs to somehow enter the .ts translation file. This is normally done by the lupdate tool, which recognizes translatable string literals as those string literals, that are either directly passed to tr() (or its equivalents) or marked with  QT_TRANSLATE_NOOP (or its equivalents). Unless we want to edit the translation file manually, we need to organize things so that lupdate can handle them, i.e., we need to use QT_TRANSLATE_NOOP somehow.

Reply:
With generated sfdk_version_p.h QT_TRANSLATE_NOOP  dosn't work yes.  So need to transfer back to h file `sfdkconstants.h` or leave as is, e.g. for Sailfish OS no translation and for iur brand we any way use some path to add translation